cmd/updater/update.sh

```diff
@@ -367,17 +367,23 @@ function run_systemd_action() {
     local data_dir=$2
     local process_owner
 
+    # If the service is system-level, check if it's root or sudo
     if check_service system "$data_dir"; then
         process_owner=$(awk '{ print $1 }' <(ps aux | grep "[a]lgod -d ${data_dir}"))
-        if $IS_ROOT || grep sudo <(groups "$process_owner" &> /dev/null); then
+        if $IS_ROOT; then
             if systemctl "$action" "algorand@$(systemd-escape "$data_dir")"; then
                 echo "systemd system service: $action"
```

Comment:
Should the echo reflect the full contents of the command above?

Reply:
I'm not sure if it should or not? It might make more sense to.